lib/Elastica/Request.php

```diff
@@ -159,6 +163,24 @@ public function getConnection()
     }
 
     /**
+     * Set the Content-Type of this request
+     * @param string $contentType
+     */
+    public function setContentType($contentType)
+    {
+        $this->setParam('contentType', $contentType);
```

Comment:
`setParam(..)` should already return $this so you can put the return in front of the call directly.